docs/src/Callout.doc.js

```diff
@@ -22,168 +47,346 @@ card(
+      <MainSection.Card
+        cardSize="lg"
+        type="do"
+        description="Use Callouts for messages coming from the product or user interaction."
+        defaultCode={`
+          <Callout
+            type="error"
+            title="Pinterest tag needs attention"
+            iconAccessibilityLabel="Error"
+            message="Your tag has errors, so information may be outdated. Fix your tag for the most accurate metrics."
+            primaryAction={{href: "https://pinterest.com", label:"Fix tag"}}
+
+          />
+        `}
+      />
+      <MainSection.Card
+        cardSize="lg"
+        type="do"
+        description={`
+        Place the Callout at the top of the page under the primary navigation or page header when possible.
+        `}
+        defaultCode={`
+          <Box>
+            <Box marginBottom={4} display="flex" alignItems="center">
+              <Icon accessibilityLabel="" icon="pinterest" color="red" size={32}/>
+              <ButtonGroup>
+                <Button color="transparent" iconEnd="arrow-down" text="Business" inline />
+                <Button color="transparent" iconEnd="arrow-down" text="Create" inline />
+                <Button color="transparent" iconEnd="arrow-down" text="Analytics" inline />
+                <Button color="transparent" iconEnd="arrow-down" text="Ads" inline />
+              </ButtonGroup>
+            </Box>
+            <Divider/>
+            <Box marginTop={8}>
+              <Callout
+                type="info"
+                iconAccessibilityLabel="Info"
+                title="We have not yet detected your tag"
+                message="It may take up to 10 minutes to automatically detect a newly installed tag. If you'd like to manually verify your tag, please click the Verify Tag button."
+                primaryAction={{label: "Verify Tag"}}
+                dismissButton={{
+                  accessibilityLabel: 'Dismiss this banner',
+                  onDismiss: ()=>{},
+                }}
+              />
+            </Box>
+          </Box>
+        `}
+      />
+      <MainSection.Card
+        cardSize="lg"
+        type="don't"
+        description={`
+        Use Callouts for marketing new products or features. Use [Upsell](/Upsell) instead.
+        `}
+        defaultCode={`
+          <Callout
+            type="info"
+            iconAccessibilityLabel="Info"
+            title="Give $30, get $60 in ads credit"
+            message="Earn $60 of ads credit, and send $30 of ads credit to a friend"
+            primaryAction={{label: "Send invite"}}
+            dismissButton={{
+              accessibilityLabel: 'Dismiss this banner',
+              onDismiss: ()=>{},
+            }}
+          />
+        `}
+      />
+
+      <MainSection.Card
+        cardSize="lg"
+        type="don't"
+        description={`
+        Stack Callouts. In the case that banners must be stacked, Callouts will usually come before Upsells.
```

Comment:
I wonder if `will usually` is confusing.  What about `Callouts come before Upsells`